src/js/components/awardv2/shared/overview/RelatedAwards.jsx

```diff
@@ -42,47 +43,67 @@ export default class RelatedAwards extends React.Component {
         this.props.setRelatedAwardsTab('grandchild_awards');
         this.props.jumpToSection('referenced-awards');
     }
+    jumpToAwardHistoryTableSubAwardsTab = () => {
+        this.props.jumpToSubAwardHistoryTable('subaward');
+        this.props.jumpToSection('award-history');
+    }
 
     referencedAwardCounts() {
-        const { counts } = this.props;
+        const { counts, overview } = this.props;
         if (!counts) return null;
-        const childData = [
-            {
-                count: formatNumber(counts.child_awards),
-                name: 'Child Award',
-                funcName: 'ChildAwards',
-                glossary: 'contract'
-            },
-            {
-                count: formatNumber(counts.child_idvs),
-                name: 'Child IDV',
-                funcName: 'ChildIDVs',
-                glossary: 'IDV'
-            },
-            {
-                count: formatNumber(counts.grandchild_awards),
-                name: 'Grandchild Award',
-                funcName: 'GrandchildAwards',
-                glossary: 'award'
-            }
-        ];
-
+        let childData = [];
+        if (overview.category === 'idv') {
+            childData = [
```

Comment:
Ideally, this would be in a datamapping file, where we would import it and map over to add `count` and `postText` properties. Not a huge deal we're doing it this way though due to the use of dynamic values for these properties.

Defer to your judgment / preference as to whether we do this.